notebooks/pytorch/Text Classification Using PyTorch and CMLE.ipynb

```diff
@@ -787,12 +786,11 @@
     "\n",
     "from setuptools import setup\n",
     "\n",
-    "REQUIRED_PACKAGES = ['torch', 'keras']\n",
+    "REQUIRED_PACKAGES = []\n",
```

Comment:
Can we remove this entirely if it's empty?

Reply:
@ksalama It appears that there are some changes being requested.

Reply:
Done

Reply:
Done